pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTest.java

```diff
@@ -517,7 +518,7 @@ public void testResetCursorNotFail() throws Exception {
 
         consumer1.receive(2);
 
-        admin1.topics().resetCursor(dest.toString(), "sub-id", System.currentTimeMillis());
+        admin1.topics().resetCursor(dest.toString(), "sub-id", MessageId.latest);
```

Comment:
We can reset the cursor by time, if the original test has some issue, is this means we have some issues on the reset cursor by time?

Reply:
+1, this is not fixing the problem, only avoiding it.
Looking at the [issue](https://github.com/apache/pulsar/issues/9450), test usually failed because 
```
04:51:29.271 [pulsar-io-59-1] WARN  org.apache.pulsar.client.impl.BinaryProtoLookupService - [persistent://pulsar/partitionedNs-false/partTopic-1613105459006-false] failed to get Partitioned metadata : 145 lookup request timedout after ms 30000
org.apache.pulsar.client.api.PulsarClientException$TimeoutException: 145 lookup request timedout after ms 30000
```
probably should dig into the look up logic to figure out why the topic is gone from broker during test.

Reply:
yes, i think the ci env has resource problem which can cause gc or pause to make TimeoutException.
will close this pr :- )